test/adapters/adapters.spec.js

```diff
@@ -193,4 +250,37 @@ describe('adapters', () => {
         });
     });
 
+    _.each([
+        'mysql',
+        'postgres',
+        'sqlite'
+    ], (adapterName) => {
+        describe(`juttle-${adapterName}-adapter`, () => {
+            it('can write data and read back data using the juttle CLI', () => {
+                var id = uuid.v1().slice(0, 8);
+                var table = `test_${id}`;
+
+                // create a new table each time
```

Comment:
The table name doesn't really have to be unique does it? You're just writing to it once.